povm_toolbox/post_processor/povm_post_processor.py

```diff
@@ -69,6 +72,9 @@ def __init__(
                 :meth:`get_decomposition_weights`. When this is ``None``, the standard
                 "state-average" Dual frame will be constructed from the POVM stored in the
                 ``povm_sample``'s :attr:`.POVMPubResult.metadata`.
+            combine_counts: indicates, when applicable, whether to combine the counts associated
+                with different parameter values that were submitted for a single parametrized
+                circuit.
```

Comment:
I think this explanation has to be extended further:
- we should clarify what "when applicable" means
- we should clarify _that_ and _how_ we use `np.ndarray.flatten`
- we should have a clear example what this implies for 0d-, 1d-, and Nd- arrays of parameters in the original job

We can either do that in the docs here, or in a how-to guide and link to that from here (which may be better as it gives us more space for explanations).